pkg/plugin/plugin.go

```diff
@@ -57,11 +57,12 @@ func (d *DataSetDatasource) QueryData(ctx context.Context, req *backend.QueryDat
 }
 
 type queryModel struct {
-	Expression          string  `json:"expression"`
-	QueryType           string  `json:"queryType"`
-	Format              string  `json:"format"`
-	BreakDownFacetValue *string `json:"breakDownFacetValue"`
-	Label               *string `json:"label"`
+	Expression          string    `json:"expression"`
+	QueryType           string    `json:"queryType"`
+	Format              string    `json:"format"`
+	BreakDownFacetValue *string   `json:"breakDownFacetValue"`
+	Label               *string   `json:"label"`
+	AccountEmails       *[]string `json:"accountEmails"`
```

Comment:
Slices (ie `[]<type>`) are reference types so you can assign nil to them inherently
```
	var a []int = []int{1, 2, 3}
	var b []int = []int{}
	var c []int = nil
```

Better to have the AccountEmails type to be just `[]string`